packages/airnode-admin/src/cli.ts

```diff
@@ -509,8 +509,8 @@ yargs
     }
   )
   .command(
-    'generate-mnemonic',
-    'Generates a random mnemonic. Uses "ethers.Wallet.createRandom" under the hood',
+    'generate-airnode-mnemonic',
+    'Generates a random mnemonic which can be used in the Airnode config and displays the corresponding address and extended public key. Uses "ethers.Wallet.createRandom" under the hood',
```

Comment:
(subjective nit)

```suggestion
    'Generates a random mnemonic which can be used in the Airnode config and displays the extended public key. Uses "ethers.Wallet.createRandom" under the hood',
```

The correspodning address sounds wierd. We could replace "corresponding" with "default". Feel free to disregard.

Reply:
We use `default wallet...` for the other `generate-mnemonic` command so perhaps just leave it out, although the current wording is how the api3-docs puts it :thinking: